crates/chia-puzzles/src/puzzles/nft.rs

```diff
@@ -132,33 +132,33 @@ pub struct NftOwnershipLayerSolution<I> {
 pub struct NftRoyaltyTransferPuzzleArgs {
     pub singleton_struct: SingletonStruct,
     pub royalty_puzzle_hash: Bytes32,
-    pub trade_price_percentage: u16,
+    pub royalty_percentage: u16,
```

Comment:
somewhat unrelated, but my recollection is that this is expressed in thousands, right? if so it should really be called `royalty_permille`